pyproject.toml

```diff
@@ -16,7 +16,6 @@ pyarrow="^19.0.1"
 pytest= "^8.3.5"
```

Comment:
Let's create a file pytest.ini in the root folder so that pytest know where to find the tests.